app/src/main/java/com/example/playlistmaker/player/AudioPlayerActivity.kt

```diff
@@ -106,6 +124,74 @@ class AudioPlayerActivity: AppCompatActivity() {
 
```

Comment:
В методе выше студия подсказывает "Use of setter method instead of property access syntax ", предпочтительней в Kotlin использовать второй вариант, он выглядит так:  nameActor.text = track.artistName